docs/tutorials/committing-liquidity.md

```diff
@@ -0,0 +1,143 @@
+
+API LINK
+```bash
+```
+
+## Creating a liquidity commitment
+This tutorial describes how to create, amend or cancel, and send a liquidity commitment submission.
+
+**The liquidity commitment submission must include**:
+* The **market’s unique ID**, denoted as `marketId` - Confirm that the market is in a state to accept liquidity commitment, and is not a rejected market, has not had trading terminated, or has not been settled 
+* **Liquidity commitment amount**: The amount of funds that you want to allocate to providing liquidity. The amount will be moved into a bond account during the duration of your liquidity commitment, denoted as `commitmentAmount`
+* **Proposed liquidity fee**: The scaling factor for the fee you are bidding to receive when your order is matched, on a scale between 0 and 1. For example, a fee level of 0.01 would mean `0.01 * total trade amount` is charged. Note: Your proposed fee is used along with other proposed fees and commitments to determine the actual fee percentage for the market. [Learn how all proposed fee levels influence the market's fees]). Denoted as `fee` Denoted as `fee`
+
+**To submit the liquidity commitment message, you'll also need**: 
+
+* Public key: The public key being used to place the liquidity commitment
+* Propagate: Is true or false. Propagate is used to define if you want the liquidity commitment sent to the nodes for processing immediately (true), or if you want to manually submit the orders in a transaction (false). Note: If you choose to manually submit, it must be within the block tolerance level or it will be rejected
+
+Note: The **commitment amount** is interpreted by the asset decimal place, and expressed in an integer. In the example below, if the asset has a decimal precision of 2, the commitment amount would be 100.
+
+
+```python
+submission = {
+    "liquidityProvisionSubmission": 
+    {
+        "marketId": marketID,
+        "commitmentAmount": "10000",              
+        "fee": "0.01",
+       },
+    "pubKey": pubkey,
+    "propagate": True
+}
+```
+## Tactics for providing liquidity [WIP]
+Providing liquidity can be done using: 
+* Standard limit orders, which give you the most control over your strategy. The [batch orders transaction](../concepts/trading-on-vega/orders#batch-order) is designed to enable this efficiently
+* [Iceberg orders](../concepts/trading-on-vega/orders.md#iceberg-order) allow LPs to remain competitively present on the order book without needing to supply excessive volume to a large aggressive order
+
+Anyone that supplies limit orders is eligible to receive maker fees when volume they place on the book is hit. However, a liquidity commitment also makes an LP eligible to receive a portion of the liquidity fee from every trade in the market, on top of the maker fee.
+
+Liquidity providers need to have enough available assets to cover the margin for their orders and the positions that will be generated from trades.
```

Comment:
```suggestion
Liquidity providers need to have enough available assets within their general and margin accounts to cover the margin for their orders and the positions that will be generated from trades. The funds within the bond account are held separate unless the combined general and margin accounts are insufficient, at which point funds from the bond account may be used. However this may come with an associated penalty.
```